From 0b1ea895081670fdca905421df583aef739c48b7 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:39 -0400
Subject: [redhat] kabi: Adding symbol proc_create_data (fs/proc/generic.c)

Message-id: <284dfc3c1c4be57a6725ab58f3d05e61412399d6.1410510773.git.poros@redhat.com>
Patchwork-id: 96080
O-Subject: [PATCH RHEL7.1 159/233] kabi: Adding symbol proc_create_data (fs/proc/generic.c)
Bugzilla: 1062361 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1062361,1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Used to create an entry in the proc filesystem with the given name
    and associated file_operations function pointers.
    SFHA creates files in the procfs to report filesystem usage statistics and runtime tuning of filesystem.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/proc_create_data
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/proc_create_data
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/proc_create_data

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/proc_create_data b/redhat/kabi/kabi-module/kabi_ppc64/proc_create_data
new file mode 100644
index 0000000..05d8424
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/proc_create_data
@@ -0,0 +1,2 @@
+#1-
+0x750498fa proc_create_data vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/proc_create_data b/redhat/kabi/kabi-module/kabi_s390x/proc_create_data
new file mode 100644
index 0000000..f2a5cdb
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/proc_create_data
@@ -0,0 +1,2 @@
+#1-
+0xf8ca2b5a proc_create_data vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/proc_create_data b/redhat/kabi/kabi-module/kabi_x86_64/proc_create_data
new file mode 100644
index 0000000..d0423d7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/proc_create_data
@@ -0,0 +1,2 @@
+#1-
+0x8c34c149 proc_create_data vmlinux EXPORT_SYMBOL
-- 
1.7.1